lib/dest/options.js

```diff
@@ -36,7 +32,7 @@ var config = {
   // Symlink options
   useJunctions: {
     type: 'boolean',
-    default: isWindows,
+    default: true,
```

Comment:
Should add inline comment about this being ignored on non-Windows (as noted in the docs)

Reply:
Makes sense, yep.